.github/workflows/continuous-benchmarking-baseline.yml

```diff
@@ -111,8 +111,12 @@ jobs:
         run: tar --strip-components=1 -xvf ../build.tar -C .
 
       - name: AWS Warm Function Invocation - Baseline
-        working-directory: ${{ env.working-directory }}
-        run: ./stellar -a 356764711652 -o latency-samples -c ../continuous-benchmarking/experiments/warm-function-invocations/warm-baseline-aws.json -db -w
+        uses: nick-fields/retry@v2
+        with:
+          timeout_minutes: 1440
```

Comment:
Is the timeout_minutes 1440 for all 5 retries?

Reply:
Yes it should be

Reply:
Cool